EIPS/eip-3102.md

```diff
@@ -0,0 +1,190 @@
+---
+eip: 3102
+title: Binary trie structure
+author: Guillaume Ballet (@gballet), Vitalik Buterin (@vbuterin)
+status: Draft
+discussions-to: https://ethresear.ch/t/binary-trie-format/7621
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-01
+requires: None
+replaces: None
```

Comment:
```suggestion
```
Just leave them out if there are no dependencies.  Otherwise the bot will complain (as it is).